### DISCUSSION TITLE
YOLO

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#44] [Q&A] [answered]
Kod düzeltmeden Pull Request oluşturdun ve kabul edildi.

**Comment by 0xpellegrino** [comment, marked as answer]
Eline sağlık hocam.

**Reply by enzifiri** [reply]
3. Başarımı tekrardan yapmayı dene hocam kodda sorun vardı düzelttim

**Comment by 0xik** [comment]
sagolun hocam
